meilisearch/src/analytics/segment_analytics.rs

```diff
@@ -220,9 +220,8 @@ struct Infos {
     ignore_missing_dump: bool,
     ignore_dump_if_db_exists: bool,
     import_snapshot: bool,
-    schedule_snapshot: bool,
+    schedule_snapshot: Option<Option<u64>>,
```

Comment:
I might have missed it, but it looks like the variant `Some(None)` is never used, could the type of this field be simplified to `Option<u64>` or maybe a new custom enum?